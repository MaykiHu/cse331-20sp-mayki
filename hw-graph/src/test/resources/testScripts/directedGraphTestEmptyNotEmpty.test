#Create a graph
CreateGraph graph1

#Add a node to graph
AddNode graph1 n1

#Checks if empty, when it isn't empty
IsEmpty graph1