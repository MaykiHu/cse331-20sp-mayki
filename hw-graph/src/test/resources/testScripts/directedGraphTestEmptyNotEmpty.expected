#Create a graph
created graph graph1

#Add a node to graph
added node n1 to graph1

#Checks if empty, when it isn't empty
graph1 is not empty